sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/azure/eventhub/extensions/checkpointstoreblobaio/_vendor/storage/blob/_download.py

```diff
@@ -656,7 +656,7 @@ def read(self, size: int = -1, *, chars: Optional[int] = None) -> T:
         output_stream: Union[BytesIO, StringIO]
         if self._text_mode:
             output_stream = StringIO()
-            size = chars if chars else sys.maxsize
+            size = sys.maxsize if chars is None or chars <= 0 else chars
```

Comment:
did mypy complain about this line? `if chars` should have evaluated to `True` if it was >= 1 or was not None

Reply:
it did not :)

Reply:
oh, then why the change ?

Reply:
this is from storage copy/paste over